compiler/src/iree/compiler/Codegen/Passes.td

```diff
@@ -235,6 +235,13 @@ def LLVMCPULowerExecutableTarget :
       "mlir::iree_compiler::createLLVMCPULowerExecutableTargetPass()";
 }
 
+def LLVMCPULowerTargetDependentTosa :
```

Comment:
Nit: I think pass can just be call `LLVMCPULowerTosaOpsPass`.  IIUC `tosa.apply_rescale` is the only op here.